pymchelper/input_output.py

```diff
@@ -156,15 +163,29 @@ def frompattern(pattern: str, error: ErrorEstimate = ErrorEstimate.stderr, nan:
     core_names_dict = group_input_files(list_of_matching_files)
 
     result = [fromfilelist(filelist, error, nan) for _, filelist in core_names_dict.items()]
+
     return result
 
 
+def get_topas_estimators(output_files_path: str) -> List[Estimator]:
+    """Get Topas estimators from provided directory"""
+    estimators_list = []
+    for result_file_path in Path(output_files_path).iterdir():
```

Comment:
```suggestion
    for path in Path(output_files_path).iterdir():
```

No reason to call all files and directories a results_file_path